ui/index.d.ts

```diff
@@ -1097,6 +1097,34 @@ export function createAzureSubscriptionClient<T extends IAddUserAgent>(
  */
 export function createApiProvider(azExts: AzureExtensionApi[]): AzureExtensionApiProvider;
 
+export interface IAzExtOutputChannel extends OutputChannel {
+    appendLog(value: string, options?: { resourceName?: string, date?: Date }): void;
+}
+/**
+ * Wrapper for vscode.OutputChannel that handles AzureExtension behavior for outputting messages
+ */
+export declare class AzExtOutputChannel implements OutputChannel {
```

Comment:
Since you're exporting the interface above, it's redundant to also export this class. The only difference is the interface can't have a constructor. To fix that, just export a function modeled after `window.createOutputChannel`